src/library-aux/scala/Singleton.scala

```diff
@@ -0,0 +1,17 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___     Scala API                            **
+**    / __/ __// _ | / /  / _ |    (c) 2002-2013, LAMP/EPFL             **
+**  __\ \/ /__/ __ |/ /__/ __ |    http://scala-lang.org/               **
+** /____/\___/_/ |_/____/_/ | |                                         **
+**                          |/                                          **
+\*                                                                      */
+
+package scala
+
+/** Scala has concept of object.An object is a class that has exactly one instance. It is created lazily.
```

Comment:
"has concept" should be "has a concept"

missing space in "object.An"

"An object is a class" seems confusing to me. Perhaps "The object's class has exactly one instance."